package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "codemirror-contrib",
-  "version": "0.0.0",
+  "version": "0.0.1",
   "description": "CodeMirror community contributions",
   "scripts": {
-    "build": "rollup --format umd -n CodeMirror src/**/*.js -o lib",
+    "build": "rollup --format umd -n CodeMirror src/mode/meta.js -o lib/contrib.js",
```

Comment:
Yeah, I don't think the glob actually worked, but I wasn't sure why. Wondering if theres any way to avoid hard coding the src list here.